Patches/Forsakens/Forsakens_CE_Patch_ThingDefs_Weapons.xml

```diff
@@ -0,0 +1,561 @@
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="ForsakenLongBlade"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>handle</label>
+								<capacities>
+									<li>Blunt</li>
+								</capacities>
+								<power>2</power>
+								<chanceFactor>0.33</chanceFactor>
+								<cooldownTime>1.52</cooldownTime>
+								<armorPenetrationBlunt>0.500</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>point</label>
+								<capacities>
+									<li>Stab</li>
+								</capacities>
+								<power>18</power>
+								<cooldownTime>1</cooldownTime>
+								<armorPenetrationBlunt>11.250</armorPenetrationBlunt>
+								<armorPenetrationSharp>90</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Point</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>edge</label>
+								<capacities>
+									<li>Cut</li>
+								</capacities>
+								<power>46</power>
+								<cooldownTime>0.5</cooldownTime>
+								<armorPenetrationBlunt>45</armorPenetrationBlunt>
+								<armorPenetrationSharp>80</armorPenetrationSharp>
+								<linkedBodyPartsGroup>Edge</linkedBodyPartsGroup>
+							</li>
+						</tools>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="ForsakenLongBlade"]/statBases</xpath>
+					<value>
+						<Bulk>8</Bulk>
+						<MeleeCounterParryBonus>1.6</MeleeCounterParryBonus>
+					</value>
+				</li>
+
+				<li Class="PatchOperationAdd">
+					<xpath>Defs/ThingDef[defName="ForsakenLongBlade"]</xpath>
+					<value>
+						<equippedStatOffsets>
+							<MeleeCritChance>4</MeleeCritChance>
+							<MeleeParryChance>1.2</MeleeParryChance>
+							<MeleeDodgeChance>0.4</MeleeDodgeChance>	
+						</equippedStatOffsets>
+					</value>
+				</li>
+
+				<!-- ========== Forsaken Combat Blade ========== -->
+
+				<!-- (CE implementation based on vanilla CE knife with armor penetration of CE Melee Wraithblade) -->
+				<li Class="PatchOperationReplace">
+					<xpath>Defs/ThingDef[defName="ForsakenCombatBlade"]/tools</xpath>
+					<value>
+						<tools>
+							<li Class="CombatExtended.ToolCE">
+								<label>handle</label>
+								<capacities>
+									<li>Poke</li>
+								</capacities>
+								<power>1</power>
+								<cooldownTime>1.26</cooldownTime>
+								<armorPenetrationBlunt>0.250</armorPenetrationBlunt>
+								<linkedBodyPartsGroup>Handle</linkedBodyPartsGroup>
+							</li>
+							<li Class="CombatExtended.ToolCE">
+								<label>blade</label>
+								<capacities>
+									<li>Cut</li>
+								</capacities>
+								<power>10</power>
+								<cooldownTime>1.18</cooldownTime>
+								<armorPenetrationBlunt>45</armorPenetrationBlunt>
+								<armorPenetrationSharp>80</armorPenetrationSharp>
```

Comment:
See the above.

Reply:
Same rebalanced sharp/blunt AP stats as above, but with even lower weapon power (10->5) - requiring more strikes to do the same amount of damage compared to the long blade.

Fixed in 512216f